README.md

```diff
@@ -35,6 +35,21 @@ release a new version, update the version number in `version.rb`, and then run
 git commits and tags, and push the `.gem` file to
 [rubygems.org](https://rubygems.org).
 
+### Copying files to Docker Container
```

Comment:
The "Development" section is more for people who are trying to contribute to the library rather than those who are trying to use it. It's better for this to go under "Usage".